Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -428,11 +452,14 @@ protected class ClassWriter : IClassWriter {
     }
 
     protected override ConcreteSyntaxTree CreateLabeledCode(string label, bool createContinueLabel, ConcreteSyntaxTree wr) {
-      throw new NotImplementedException();
+      Contract.Assert(!createContinueLabel);
+      return wr.NewBlockPy($"with _dafny.label(\"{label}\"):");
     }
 
     protected override void EmitBreak(string label, ConcreteSyntaxTree wr) {
-      throw new NotImplementedException();
+      if (label != null) {
+        wr.WriteLine($"_dafny._break(\"{label}\")");
+      } else { wr.WriteLine("break"); }
```

Comment:
```suggestion
      } else {
        wr.WriteLine("break");
      }
```
Just a nit. Same-line blocks are fine in one liners but I find I trip over them when an if chain mixes and matches.